README.md

```diff
@@ -74,6 +74,47 @@ Building python pipeline and (optional) Setting up virtual environment
    * To install to develop: "python3 setup.py develop"
    * To build a source distribution: "python3 setup.py sdist"
 
+Postprocessing traits for manual curation
```

Comment:
This section should be part of the wiki, not the README file, because it consists of steps needed for submitting to OpenTargets.

Reply:
For the purposes of _this_ pull request, I moved the instructions to the wiki. I will provide another pull request with large update & restructuring of documentation.